File: KMS/KMS-war/src/java/ws/restful/model/ProjectRecommendationBasedOnFollowingRsp.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package ws.restful.model;

import entity.ProjectEntity;
import entity.UserEntity;
import java.util.List;

/**
 *
 * @author Cassie
 */
public class ProjectRecommendationBasedOnFollowingRsp {
    
    private ProjectEntity projectToRecommend;
    
    private List<UserEntity> followingInProject;

    public ProjectRecommendationBasedOnFollowingRsp() {
    }

    public ProjectEntity getProjectToRecommend() {
        return projectToRecommend;
    }

    public void setProjectToRecommend(ProjectEntity projectToRecommend) {
        this.projectToRecommend = projectToRecommend;
    }

    public List<UserEntity> getFollowingInProject() {
        return followingInProject;
    }

    public void setFollowingInProject(List<UserEntity> followingInProject) {
        this.followingInProject = followingInProject;
    }
}

File: KMSIonic/src/app/models/ProjectRecommendationBasedOnFollowingRsp.ts
import { Project } from '../classes/project';
import { User } from '../classes/user';

export class ProjectRecommendationBasedOnFollowingRsp {
  projectToRecommend: Project;
  followingInProject: User[];

  constructor(projectToRecommend?: Project, followingInProject?: User[]) {
    this.projectToRecommend = projectToRecommend;
    this.followingInProject = followingInProject;
  }
}
